src/components/organisms/deployments/table.tsx

```diff
@@ -91,11 +92,11 @@ export const DeploymentsTable = () => {
 
 			{loadingDeployments && isInitialLoad ? <Loader isCenter size="xl" /> : null}
 
-			{!loadingDeployments && !deployments?.length ? (
+			{!deployments?.length ? (
```

Comment:
```suggestion
			{!isInitialLoad && !deployments?.length ? (
```

To prevent flickering.
We don't want to display it before the first load.

Reply:
changed